core/src/main/java/hudson/model/AdministrativeMonitor.java

```diff
@@ -201,13 +205,50 @@ public Permission getRequiredPermission() {
         return Jenkins.ADMINISTER;
     }
 
+    /**
+     * Checks if the current user has the minimum required permission to view this administrative monitor.
+     * <p>
+     * Subclasses may override this method and {@link #hasRequiredPermission()} instead of {@link #getRequiredPermission()} to perform more complex permission checks,
+     * for example, checking either {@link Jenkins#MANAGE} or {@link Jenkins#SYSTEM_READ}.
```

Comment:
The Javadoc is misleading as it implies that you could grant permission to see admin monitors based on some other criteria. But in fact due to `hasPermissionToDisplay` these are really the only permissions you can check.